src/main/java/org/terasology/launcher/repositories/JenkinsClient.java

```diff
@@ -73,8 +82,8 @@ Properties requestProperties(final URL artifactUrl) {
             final Properties properties = new Properties();
             properties.load(inputStream);
             return properties;
-        } catch (IOException e) {
-            e.printStackTrace();
+        } catch (IOException | URISyntaxException | InterruptedException e) {
```

Comment:
Why do we catch the `InterruptedException` here but not in the other places?

Reply:
oh, because it was used in Optional.map, and the type of thing Optional.map takes cannot throw checked exceptions